dateparser/timezones.py

```diff
@@ -6,7 +6,7 @@
 timezone_info_list = [
     {
         'regex_patterns':
-            [r'(\W|\d|_)\(?%s\)?$'],
+            [r'(\W|\d|_)\(?%s(\)|$|[^\w+])'],
```

Comment:
it checks for three possibilities: Is the character next to timezone abbreviation e.g. `EDT` a closing bracket? or is end of the string? or is *not* any word character [a-zA-Z0-9_]